homeassistant/components/sochain/sensor.py

```diff
@@ -1,4 +1,5 @@
 """Support for watching multiple cryptocurrencies."""
+# pylint: disable=import-error
```

Comment:
Not critical but should we do it just on the `from pysochain import ChainSo`, looking at this I remembered this is what we did in the past when disabling integration

Reply:
I think it's ok, it is disabled. This PR can be reverted when the integration is enabled again.